andela/ah-frontend-thor
# [ft #162572369]Add view all and one article

#### What does this PR do?
Enables a user to be able to view many or a particular article.

#### Description of Task to be completed?
Users should be able to view one or many articles.

#### How should this be manually tested?
 - Clone the repository.
 - Install application dependencies using`yarn`
 - start the server using `yarn start:dev`
 - Enter `localhost:8080/` in the browser
 - A user should be able to view all the articles created by other authors and on click on one of them, they should be able to view the details of that particular article.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#162572369
